.github/CODEOWNERS

```diff
@@ -3,37 +3,61 @@
 
 # Wasm Runtime
 packages/wasm/as  @dOrgJelli @krisbitney @namesty @Niraj-Kamdar
+packages/wasm/rs  @namesty @dOrgJelli
+
+# Wrapper Test-Cases
+packages/test-cases/cases/wrappers/wasm-as @cbrzn @namesty @nerfZael @krisbitney @Niraj-Kamdar
+packages/test-cases/cases/wrappers/wasm-rs @cbrzn @namesty @nerfZael @krisbitney @Niraj-Kamdar
 
 # Schema
 packages/schema/parse    @dOrgJelli @namesty @nerfZael @Niraj-Kamdar
 packages/schema/compose  @dOrgJelli @namesty @nerfZael @Niraj-Kamdar
-packages/schema/bind     @dOrgJelli @krisbitney @nerfZael @Niraj-Kamdar
+packages/schema/bind     @dOrgJelli @krisbitney @nerfZael @Niraj-Kamdar @cbrzn
+
+# Schema Test-Cases
+packages/test-cases/cases/parse    @dOrgJelli @namesty @nerfZael @Niraj-Kamdar
+packages/test-cases/cases/compose  @dOrgJelli @namesty @nerfZael @Niraj-Kamdar
+packages/test-cases/cases/bind     @dOrgJelli @krisbitney @nerfZael @Niraj-Kamdar @cbrzn
 
 # Project Manifests
-packages/manifest-schemas/  @dOrgJelli
+packages/manifests/  @dOrgJelli @namesty @cbrzn @Niraj-Kamdar
 
 # CLI
 packages/js/cli @dOrgJelli @krisbitney @namesty @nerfZael @Niraj-Kamdar
 
-# Client Interfaces
-packages/interfaces  @dOrgJelli
+# CLI Test-Cases
+packages/test-cases/cases/cli @dOrgJelli @krisbitney @namesty @nerfZael @Niraj-Kamdar
+
+# Interfaces
+packages/interfaces  @dOrgJelli @krisbitney @cbrzn
 
 # JS Client
-packages/js/client    @dOrgJelli @krisbitney @namesty @nerfZael
-packages/js/react     @dOrgJelli @nerfZael
-packages/js/core      @dOrgJelli @krisbitney @namesty @nerfZael
-packages/js/os        @dOrgJelli
-packages/js/asyncify  @namesty @dOrgJelli
-packages/js/tracing   @dOrgJelli @Niraj-Kamdar
-packages/js/test-env  @dOrgJelli
+packages/js                          @dOrgJelli @nerfZael
+packages/js/asyncify                 @namesty @dOrgJelli
+packages/js/client                   @dOrgJelli @krisbitney @namesty @nerfZael
+packages/js/client-config-builder    @nerfZael
```

Comment:
I might be a good candidate for the `client-config-builder` package too.

```CODEOWNERS
packages/js/client-config-builder @nerfZael @pileks
```

Reply:
Oh wow, thought I had you, weren't you the one that created the package in the first place?

It looks like the commit history was cut off for some reason, looks like a bad merge on my part. My apologies 🙏 
https://github.com/polywrap/toolchain/commits/origin-0.10-dev?after=7c7d471d6bf23c4464fdd3510a359690be86ed28+34&branch=origin-0.10-dev&path%5B%5D=packages&path%5B%5D=js&path%5B%5D=client-config-builder&qualified_name=refs%2Fheads%2Forigin-0.10-dev